### DISCUSSION TITLE
NodeJS sidecar for mobile apps?

### DISCUSSION THREAD

**Discussion by trymeouteh** [tauri-apps/tauri#11454] [Q&A]
Is it possible to compile backend JS code such as NodeJS or Deno into a binary and be able to use it as a sidecar for Tauri Android and iOS apps? The guide in the Tauri docs do not explain how to use the sidecar on mobile, only desktop.

**Comment by FabianLars** [comment]
Not yet, no. I think this is the issue we use to track mobile support: https://github.com/tauri-apps/tauri/issues/9774
